src/status_im/common/floating_button_page/view.cljs

```diff
@@ -52,7 +52,7 @@
      :remove-listeners    remove-listeners}))
 
 (defn view
-  [{:keys [header footer customization-color gradient-cover?]} &
+  [{:keys [header footer customization-color keyboard-suggestion-bar? gradient-cover?]} &
```

Comment:
why don't we just make this value programmatical?
ie.g `footer-container-padding` and add the safe-area as a default

Reply:
It is a valid suggestion, updated the approach!